history-of-the-quakers-v3/original/28-book-6-chapter-8.adoc

```diff
@@ -344,7 +373,7 @@ and another signed by several neighbours, persons of character and consideration
 who testified that the bible was daily read in this school (which
 was a very large boarding school kept by Richard Scoryer,
 a friend) in a regular course of succession from the beginning to the end.
-This book, entitled the Snake in the Grass, did not pass unnoticed.
+This book, entitled the [.book-title]#Snake in the Grass,# did not pass unnoticed.
 The author`'s misrepresentations were laid open, his fabulous tales disproved,
 and his crafty imposture clearly manifested in suitable
 replies by whitehead George Whitehead,
```

Comment:
can you add a book-title around `A Switch for the Snake` below? I just happened to notice it, and just in case it was not italicized and might get missed.  Thanks!